opg-nginx-1604/tests/controls/default.rb

```diff
@@ -47,3 +47,18 @@
     its(:content) { should match /"query_string": "myquerystring"/ }
   end
 end
+
+control 'Sirius ZF Matched Route Name Exists' do
+  impact 1
+  title 'Sirius ZF Matched Route Name Exists'
+  desc 'Sirius ZF Matched Route Name Exists'
+  # app severname will not log for static requests so no logs will be produced.
+  # Workaround by sending to default host which drops connection but does
+  # add 444 request to logs.
+  describe command('curl -k "https://localhost/matched_route_path?myquerystring" -H "X-Sirius-ZF-Matched-Route-Name: SomeRouteValue"') do
```

Comment:
Why not combine your test with the one above?

Reply:
done